dm/pkg/schema/tracker.go

```diff
@@ -136,6 +137,10 @@ func (tr *Tracker) Init(
 
 	logger = logger.WithFields(zap.String("component", "schema-tracker"), zap.String("task", task))
 
+	tidbConfig.UpdateGlobal(func(conf *tidbConfig.Config) {
+		conf.MaxIndexLength = 3328
```

Comment:
Can you update to a larger value, like the max int.

Reply:
ah yes, accidentally took this from our internal config, but updated to max possible value (3072*4)